### ISSUE TITLE
Uploading files to multiple site collections returns 403 forbidden

### ISSUE THREAD

**Issue by SamiSaves** [estruyf/gulp-spsync-creds#3]
Hello!

I am trying to upload files to multiple site collections within one gulp command. The first task "upload-en" goes through fine but when it gets to the second task it returns following error.

```
Unhandled rejection StatusCodeError: 403 - "{\"error\":{\"code\":\"-2130575251, System.Runtime.InteropServices.COMException\",\"message\":{\"lang\":\"en-US\",\"value\":\"The security validation for this page is invalid. Click Back in your Web browser, refresh the page, and try your operation again.\"}}}"
    at new StatusCodeError (node_modules\request-promise\lib\errors.js:32:15)
    at Request.RP$callback [as _callback] (node_modules\request-promise\lib\rp.js:77:29)
    at Request.self.callback (node_modules\request\request.js:187:22)
    at emitTwo (events.js:87:13)
    at Request.emit (events.js:172:7)
    at Request.<anonymous> (node_modules\request\request.js:1044:10)
    at emitOne (events.js:77:13)
    at Request.emit (events.js:169:7)
    at IncomingMessage.<anonymous> (node_modules\request\request.js:965:12)
    at emitNone (events.js:72:20)
    at IncomingMessage.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:921:12)
    at nextTickCallbackWith2Args (node.js:442:9)
    at process._tickCallback (node.js:356:17)

```

```
gulp.task('default', ['upload-en'], function() {
    return gulp.src(siteStructureFolders['upload-fi']).pipe(sp.sync(settings.get('/sites/fi')));
});
```

The verbose logging said that it was using cached digestVal in the second gulp task. I managed to fix the problem by resetting the digestVal variable at the start of fileSyncHelper init() function. 

Do you think the cached digestVal is causing the problem here?

**Comment by estruyf** [comment]
@Doublecaster thanks for opening an issue. Caching of the digestVal will indeed be the issue. I'll add the URL to the digestValue object to know for which URL it is cached.

**Comment by estruyf** [comment]
Just released version 2.2.5 with this fix. Can you check if it works?

**Comment by SamiSaves** [comment]
Tested the update and it fixed the problem. Thanks for the quick response and fix!
